Example_Programs/programming_concpets/ttc_1a/src/c_div.c

```diff
@@ -0,0 +1,19 @@
+#include "header_def.h"
+
+int div(complex_num s1, complex_num s2){
+    struct complex_num res;
+
+    if(s2.real == 0 || s2.img == 0 ){
+        return -1;
+        printf("You cannot divide by 0");
```

Comment:
Never gets to run. Dead code.

Reply:
I wrote this code before the template code was uploaded, I completed the template code that was pushed by you, sir. Please have a look at it.

Reply:
Cool. I'm adding comments for understanding and avoid this kind of coding.